vendor/github.com/fastly/go-fastly/v9/fastly/product_enablement.go

```diff
@@ -20,6 +20,8 @@ type Product int64
 
 func (p Product) String() string {
 	switch p {
+	case ProductBotManagement:
```

Comment:
This is a vendored copy of go-fastly and so shouldn't be modified in this repository. If the current version of go-fastly doesn't offer this product it should be added there and then the dependency version updated in this repository.

Reply:
👋🏻 @BrooksCunningham as Kevin mentioned you'll need to open a PR on this repo file:
https://github.com/fastly/go-fastly/blob/main/fastly/product_enablement.go#L21-L39

Reply:
Once a go-fastly PR has been reviewed/approved/merged, then a new release can be published and your PR here for TF can bump to that new go-fastly release.

Reply:
Thank you for the guidance! I create a PR against the go-fastly repo.

https://github.com/fastly/go-fastly/pull/539

I made an attempt at a test file as well. I have no idea if the test is working as expected/desired.

Please let me know if I may make any further updates to that PR. Or feel free to make updates :-)